entrypoint.sh

```diff
@@ -44,8 +47,12 @@ if [ "${scanType}" = "fs" ];then
   artifactRef=$(echo $scanRef | tr -d '\r')
 fi
 ignoreUnfixed=$(echo $ignoreUnfixed | tr -d '\r')
+input=$(echo $input | tr -d '\r')
 
 ARGS=""
+if [ "$input" == "true" ]; then
+  ARGS="$ARGS --input"
+fi
```

Comment:
Could you explain how this works? `--input` is a file path when it comes to Trivy as an option. 
```
 --input value, -i value     input file path instead of image name [$TRIVY_INPUT]
```

But here it's used as a boolean flag. Do you have a test repo where I can see the output?

Reply:
Updated the PR and here is the test run on alpine tar file
https://github.com/rahul2393/trivy-ci-test/runs/2183150373?check_suite_focus=true